src/mlpack/tests/main_tests/krann_test.cpp

```diff
@@ -0,0 +1,335 @@
+#include <string>
+
+#define BINDING_TYPE BINDING_TYPE_TEST
+static const std::string testName = "K-RankApproximateNearestNeighborsSearch";
+
+#include <mlpack/core.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include "test_helper.hpp"
+#include <mlpack/methods/rann/krann_main.cpp>
+
+#include <boost/test/unit_test.hpp>
+#include "../test_tools.hpp"
+
+using namespace mlpack;
+
+struct  KRANNTestFixture
```

Comment:
```suggestion
struct KRANNTestFixture
```
No need for an extra space. :)